src/stories/controls/AnyOfControl.stories.tsx

```diff
@@ -65,18 +65,18 @@ export const RadioGroup: Story = {
   },
 }
 
-export const Button: Story = {
+export const Button: Story<typeof splitterAnyOfJsonSchema> = {
   parameters: { controls: { expanded: true } },
   tags: ["autodocs"],
   args: {
-    jsonSchema: anyOfJsonSchema,
+    jsonSchema: splitterAnyOfJsonSchema,
     uiSchema: {
       type: "VerticalLayout",
       elements: [
         {
           type: "Control",
           scope: "#/properties/splitter",
-          options: { optionType: "button" } satisfies OneOfControlOptions,
```

Comment:
Why remove this? If this is meant to be an example of actual usage, does that mean we shouldn't be using this `options` `satisfies` pattern internally?

Reply:
the typechecking now happens automatically bc I modified the `Story` type to take the jsonschema type as a type argument--try changing "button" to "foo"

Reply:
I updated the readme to include the right pattern